packages/experimental/Badge/src/BadgeColorTokens.ts

```diff
@@ -1,145 +1,497 @@
+import { ColorValue } from 'react-native';
 import { Theme, TokenSettings } from '@fluentui-react-native/framework';
 import { globalTokens } from '@fluentui-react-native/theme-tokens';
 import { BadgeTokens } from './Badge.types';
 
 export const defaultBadgeColorTokens: TokenSettings<BadgeTokens> = (t: Theme) =>
   ({
     filled: {
-      backgroundColor: t.colors.brandedBackground,
-      color: t.colors.neutralForegroundInverted,
-      iconColor: t.colors.neutralForegroundInverted,
+      ...getFilledColorProps(
+        {
+          backgroundColor: t.colors.brandBackgroundStatic,
+        },
+        t,
+      ),
       borderColor: 'transparent',
     },
     outline: {
+      ...getOutlineColorProps({ color: t.colors.brandForeground1, colorDark: t.colors.brandForeground1 }, t),
       backgroundColor: t.colors.transparentBackground,
-      color: t.colors.brandForeground1,
-      borderColor: t.colors.brandForeground1,
-      iconColor: t.colors.brandForeground1,
     },
     tint: {
```

Comment:
>   tint: { [](http://example.com/codeflow?start=1&length=10)

This is still depending on globals, so it won't update to reflect the host app's brand color. Ex. If someone uses this badge in PPT, they'll get a blue badge instead.

Reply:
Yeah, but what color to use then? I checked alias tokens and there are no needed colors. I was thinking to change createAliasesFromPalette file. Tint brand colors are absent in the readlines for win32

Reply:
Talked offline and we should use the alias tokens that are specified by the redlines.